include/avif/avif.h

```diff
@@ -552,6 +564,14 @@ typedef enum avifChromaUpsampling
     AVIF_CHROMA_UPSAMPLING_BILINEAR = 4      // Uses bilinear filter (built-in)
 } avifChromaUpsampling;
 
+typedef enum avifChromaDownsampling
```

Comment:
In the current implementation, `AVIF_CHROMA_DOWNSAMPLING_AUTOMATIC` and `AVIF_CHROMA_DOWNSAMPLING_FASTEST` are the same, and `AVIF_CHROMA_DOWNSAMPLING_BEST_QUALITY` and `AVIF_CHROMA_DOWNSAMPLING_AVERAGE` are the same.

Please confirm this is the intention. Also, do you think it will stay this way?

Reply:
The intention was consistency by having flags as similar to `avifChromaUpsampling` as possible. The flags and/or behavior may change when SharpYUV and/or other libyuv functions are made available.